src/compiler/scala/tools/nsc/backend/jvm/opt/CallGraph.scala

```diff
@@ -418,13 +418,13 @@ class CallGraph[BT <: BTypes](val btypes: BT) {
     private val metafactoryHandle = {
       val metafactoryMethodName: String = "metafactory"
       val metafactoryDesc: String       = "(Ljava/lang/invoke/MethodHandles$Lookup;Ljava/lang/String;Ljava/lang/invoke/MethodType;Ljava/lang/invoke/MethodType;Ljava/lang/invoke/MethodHandle;Ljava/lang/invoke/MethodType;)Ljava/lang/invoke/CallSite;"
-      new Handle(Opcodes.H_INVOKESTATIC, lambdaMetaFactoryInternalName, metafactoryMethodName, metafactoryDesc)
+      new Handle(Opcodes.H_INVOKESTATIC, lambdaMetaFactoryInternalName, metafactoryMethodName, metafactoryDesc, /* itf = */ false)
```

Comment:
Is it possible to use CoreBTypes.jliLambdaMetafactoryRef instead of creating a new reference to j.l.i.LambdaMedatafactory. ?